README.md

```diff
@@ -79,6 +79,18 @@ other OneDrive clients:
 
 ## Quick start
 
+### Debian/Ubuntu
+
+Users of Debian (soon available via backports for Debian 12) and Ubuntu 24.04 or later
+are advised to use the official package. 
+
+Debian: https://packages.debian.org/onedriver
```

Comment:
I'm not sure these links are really needed, since most users won't probably have much of a use for them. I'd take them out.